types/popper.js/index.d.ts

```diff
@@ -1,58 +1,75 @@
-// Type definitions for popper.js v0.4.0
+// Copyright 2017 Canva Inc. All Rights Reserved.
```

Comment:
I think copyright notice should be added after other headers because some bots look `Type definitions for ...` header.